channels/channel.mtg/viafree/chn_viafree.py

```diff
@@ -495,10 +496,8 @@ def create_video_item(self, result_set):
 
             # if the name had the episode in it, translate it
             if episode > 0 and season > 0:  # and not webisode:
-                description = "%s\n\n%s" % (title, description)
-                title = "{0} - s{1:02d}e{2:02d}".format(result_set["format_title"],
-                                                        season,
-                                                        episode)
+                is_episodic = True
+                title = "{} {}".format(LanguageHelper.get_localized_string(LanguageHelper.EpisodeId), episode)
```

Comment:
How would this appear? If sorted by episode: "1x01 Episode"?. But if sorted by name or date, this will not work? As all will appear as "Episode"?

Reply:
When using the Viafree webbsite all series seems to have "Avsnitt 1", "Avsnitt 2" a.s.o as episode name and I don't think I ever seen any episode name in Retrospect for Viafree series. When I look in Retrospect series folder it's only showing "TV show title(same for every episode)" - "season/episode". So that's the reason I did it like this. I used the the localized text as it's used in several countries.

Maybe my assumtion is wrong?

Reply:
Yes, I see now.

Reply:
Seen another problem. A new series "Jordan, forensic pathologist" has just been added with 6 seasons. It seems that the episodes are not placed in a sequence, from first to last, but each season is displayed divided into several Pages. 
I think it woild be great to replace the "Page" folders and instead add season folders with sections for each season and that will also help, when sorting by name or date,  it winll not showing Episode 1, Episode , Episode 1 if there is 3 seasons.

Reply:
The paging is done server side: http://playapi.mtgx.tv/v3/videos?format=22734&type=program&order=-airdate&page=2

If you can figure out what parameters in the URL would make more sense, let me know.

Reply:
OK. Since it does not seems possible to use Season folders and that sorting can mess things up, so then it might not be a good chanage. 

Please, reject this PR.

Reply:
This change will be as good as what is used for series in the menu "TV shows" in KODI.

Reply:
Added some series to KODI TV shows and I got the following sort methods:
![sort1](https://user-images.githubusercontent.com/46671821/146560776-768e9f76-9991-4f93-9c4e-7341fcf735a8.png)
Only "Name" will remove 1x01 a.s.o. and is sorting by the name of the episode. All the other will have 1x01 a.s.o and sorted by episode and will display selected info to the right.

In Retrospect, the "Episode" sort method will show 1x01 a.s.o. and when select "Name" or "Date" the 1x01 is removed and sort by the episode name. 
To follow the KODI way, sort by the "Date" maybe should have the 1x01 a.s.o (I'm usually using "Epiosde" as Sort by, the default, so no problem I guess)

All changes done from my side.